utils/rpms/daos.spec

```diff
@@ -554,6 +554,9 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Fri Apr  8 2022 Dmitry Eremin <dmitry.eremin@intel.com> 2.3.100-2
```

Comment:
The version 2.3.100-2 is already landed. So, you should bump the version everywhere appropriately with next number.

Reply:
ok. I am going to update it. Thank you!